docs/src/pages/components/autocomplete/Virtualize.js

```diff
@@ -42,7 +42,7 @@ const ListboxComponent = React.forwardRef(function ListboxComponent(props, ref)
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
-    <div ref={ref}>
+    <div ref={ref} className={className}>
```

Comment:
I'm not sure it's better 🤔

![Capture d’écran 2019-12-10 à 18 21 43](https://user-images.githubusercontent.com/3165635/70552593-f3976700-1b79-11ea-8855-cd095d88f61a.png)

Reply:
ok, I'll revert, I was trying to bring back the padding that exists on the origonal

Reply:
It would be perfect if downshift had an offset option.